package.json

```diff
@@ -2,7 +2,7 @@
   "name": "mixpanel-browser",
   "version": "2.32.0",
   "description": "The official Mixpanel JavaScript browser client library",
-  "main": "dist/mixpanel.cjs.js",
+  "main": "dist/mixpanel.cjs.min.js",
```

Comment:
Thanks for the patch. There's certainly no harm in offering a pre-minified version of the CommonJS build alongside the other builds for those who want it, but I'm much more hesitant to make it the "main" module that gets imported when you do `require('mixpanel-browser')`. It's not typical for NPM packages to come pre-minified; it impedes dev/debugging/source-mapping and works against the usual workflow where you configure your bundler (e.g., Webpack) to run the same minifier over everything in the bundle for production.

With that in mind I'd leave this line as is and let people do `require('mixpanel-browser/dist/mixpanel.cjs.min')` if they want to do the exceptional thing of putting a pre-minified dependency into their bundles.

Reply:
Shoot, I forgot about that - you're right @tdumitrescu. With that said, I think this PR doesn't have much valuable purpose. I'm going to close it. Thanks for looking.